rust/src/util.rs

```diff
@@ -92,3 +94,35 @@ pub static RELATIVE_DISTINGUISHED_NAME: LazyPyImport =
 pub static NAME: LazyPyImport = LazyPyImport::new("cryptography.x509", &["Name"]);
 pub static DIRECTORY_NAME: LazyPyImport =
     LazyPyImport::new("cryptography.x509", &["DirectoryName"]);
+
+pub fn generate_random_bytes() -> Vec<u8> {
```

Comment:
I would rename this, since it does more than generate random bytes. Maybe something like `generate_random_bytes_for_biguint`?

Reply:
Makes sense!